rln/src/public_api_tests.rs

```diff
@@ -543,8 +535,7 @@ fn test_rln_proof() {
     }
 
     // We create a new RLN instance
-    let input_buffer =
-        Cursor::new(json!({ "resources_folder": TEST_RESOURCES_FOLDER }).to_string());
+    let input_buffer = Cursor::new(json!({}).to_string());
```

Comment:
maybe this can be abstracted away `generate_input_buf` for example, and reused in all tests